samples/emoji-search/ios/shared/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+import org.jetbrains.kotlin.gradle.plugin.PLUGIN_CLASSPATH_CONFIGURATION_NAME
+
+plugins {
+  kotlin("multiplatform")
+  kotlin("native.cocoapods")
+}
+
+kotlin {
+  ios()
+  iosSimulatorArm64("ios")
```

Comment:
I will push to your branch and fix this up. I don't know why they didn't change the `ios()` shortcut to include the new simulator target.

Reply:
It looks like this builds, though. Bonkers!

I _think_ you're overriding the shared ios source set that targets actual devices and the x64 simulator with just the arm64 simulator which is why it works

Reply:
> I don't know why they didn't change the `ios()` shortcut to include the new simulator target.

https://youtrack.jetbrains.com/issue/KT-50615/